<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="inputBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>3</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <widget class="QPushButton" name="openXMLBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>18</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Choosing a QE output xml file to parse the results.</string>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QLabel" name="directoryLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Please Select a XML File to Parse</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="outputBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>80</y>
     <width>341</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QTextEdit" name="outputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="plotBandBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>178</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Plotting the Bandgap calculated through the results. </string>
    </property>
    <property name="text">
     <string>Plot Band</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>178</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Saving the results into a file. </string>
    </property>
    <property name="text">
     <string>Save Output</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xRangeInput">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>34</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>G,X,Y,Z,G</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yRangeInput">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-1,1</string>
    </property>
   </widget>
   <widget class="QLabel" name="xRangeLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>18</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X Labels</string>
    </property>
   </widget>
   <widget class="QLabel" name="yRangeLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>93</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Range</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>178</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>View structure. </string>
    </property>
    <property name="text">
     <string>View Structure</string>
    </property>
   </widget>
   <widget class="QLabel" name="occupationLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>132</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Occupation Value:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="occupationSpin">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-10</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="xTicksCB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>208</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X Ticks</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="yTicksCB">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>208</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Ticks</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="showBandCB">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>208</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show BandGap</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="showLinesCB">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>228</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Lines</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="xRangeLabel_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>55</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X Label Locations</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xTicksLocationInput">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0,10,20,30,40</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Tools</string>
   </property>
   <widget class="QPushButton" name="dosPlotBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>Plot DOS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nebBtn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>NEB Plotter</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
